src/astro/utils/database.py

```diff
@@ -0,0 +1,82 @@
+from typing import Union
+
+from airflow.hooks.base import BaseHook
+from airflow.providers.sqlite.hooks.sqlite import SqliteHook
+from sqlalchemy import create_engine, text
+from sqlalchemy.engine import Engine
+from sqlalchemy.engine.result import ResultProxy
+
+from astro.constants import Database
+from astro.utils.dependencies import BigQueryHook, PostgresHook, SnowflakeHook
+
+
+def get_database_name(interface: Union[Engine, BaseHook, SqliteHook]) -> Database:
```

Comment:
Why do we have sqlitehook separately mentioned, since we already have BaseHook  here?